xskillscore/core/probabilistic.py

```diff
@@ -555,16 +556,20 @@ def rps(
           for ``category_edges`` for forecasts. Use this if your category edges vary
           across dimensions of forecasts and observations, and are different for each.
 
-        - None: expect than observations and forecasts are already CDFs containing
-          ``category_edge`` dimension. Use this if your category edges vary across
-          dimensions of forecasts and observations, and are different for each.
+        - None: expect than observations and forecasts are already cumulative ``cdf``
```

Comment:
Maybe break this into two sentences for clarity:
"None: expect that observations and forecasts are already cumulative distribution functions (``cdf``) or probability density functions (``pdf``), as specified by the ``category_dist`` arg. In this case, the inputs ``observations`` and ``forecasts`` must contain a dimension named ``category`` with the bin values of the density function."